hermeto/core/package_managers/oci/oci_client.py

```
@@ -0,0 +1,383 @@
+            if token:
+                headers["Authorization"] = f"Bearer {token}"
+
+            async with self._session.get(url, headers=headers) as resp:
+                if resp.status == 401 and attempt == 0 and token is not None:
+                    continue
+                if resp.status != 200:
+                    body = await resp.text()
+                    raise FetchError(
+                        f"Failed to fetch manifest for {image.repository}@{image.digest}: "
+                        f"HTTP {resp.status}: {body}"
+                    )
```

Ruff (0.15.21)

[warning] Avoid specifying long messages outside the exception class

(TRY003)